.travis.yml

```
@@ -19,36 +19,36 @@ env:
       name: checks
     - script: sbt scripted
       name: Test scripted
-    - script: sbt publishM2 && cd plugin-tester-java && mvn -Dakka.grpc.project.version=`git describe --tags | sed -e "s/v\(.*\)-\([0-9][0-9]*\).*/\\1+\\2-/"``git rev-parse HEAD | head -c8` akka-grpc:generate compile
+    - script: sbt -jvm-opts .jvmopts-travis publishM2 && cd plugin-tester-java && mvn -Dakka.grpc.project.version=`git describe --tags | sed -e "s/v\(.*\)-\([0-9][0-9]*\).*/\\1+\\2-/"``git rev-parse HEAD | head -c8` akka-grpc:generate compile
       name: Test Maven Java
-    - script: sbt publishM2 && cd plugin-tester-scala && mvn -Dakka.grpc.project.version=`git describe --tags | sed -e "s/v\(.*\)-\([0-9][0-9]*\).*/\\1+\\2-/"``git rev-parse HEAD | head -c8` akka-grpc:generate scala:compile
+    - script: sbt -jvm-opts .jvmopts-travis publishM2 && cd plugin-tester-scala && mvn -Dakka.grpc.project.version=`git describe --tags | sed -e "s/v\(.*\)-\([0-9][0-9]*\).*/\\1+\\2-/"``git rev-parse HEAD | head -c8` akka-grpc:generate scala:compile
       name: Test Maven Scala
-    - script: sbt publishM2 && cd plugin-tester-java && ./gradlew --include-build ../gradle-plugin clean compileJava
+    - script: sbt -jvm-opts .jvmopts-travis publishM2 && cd plugin-tester-java && ./gradlew --include-build ../gradle-plugin clean compileJava
       name: Test Gradle Java
-    - script: sbt publishM2 && cd plugin-tester-scala && ./gradlew --include-build ../gradle-plugin clean compileScala
+    - script: sbt -jvm-opts .jvmopts-travis publishM2 && cd plugin-tester-scala && ./gradlew --include-build ../gradle-plugin clean compileScala
       name: Test Gradle Scala
-    - script: cd benchmark-java && sbt test:compile
+    - script: cd benchmark-java && sbt -jvm-opts .jvmopts-travis test:compile
       name: Compile benchmarks
     - stage: whitesource
-      script: git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH" && sbt whitesourceCheckPolicies whitesourceUpdate
+      script: git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH" && sbt -jvm-opts .jvmopts-travis whitesourceCheckPolicies whitesourceUpdate
     - stage: publish
-      script: sbt ++2.12.10 publish ++2.13.1 "project akka-grpc-maven-plugin; set publish/skip := true; project akka-grpc-codegen; set publish/skip := true; project akka-grpc-scalapb-protoc-plugin; set publish/skip := true; project sbt-akka-grpc; set publish/skip := true; project akka-grpc" publish
-      name: sbt publish
+      script: sbt -jvm-opts .jvmopts-travis ++2.12.10 publish ++2.13.1 "project akka-grpc-maven-plugin; set publish/skip := true; project akka-grpc-codegen; set publish/skip := true; project akka-grpc-scalapb-protoc-plugin; set publish/skip := true; project sbt-akka-grpc; set publish/skip := true; project akka-grpc" publish
+      name: sbt -jvm-opts .jvmopts-travis publish
```

Too much search and replace?

```suggestion
      name: sbt publish
```

---

does it do harm there? seems simpler to have it consistent

---

oh, not in the name :D